assets/styles/nav.css

```diff
@@ -1,14 +1,25 @@
+@media only screen and (min-width: 800px) {
+  nav {
+    position: fixed;
+  }
+}
+@media only screen and (max-width: 800px) {
+  nav:after { /* doubles nav height for mobile */
+    content: "";
+    height: 35px;
+    width: 100%;
+  }
+}
```

Comment:
This is a huge hack to get the nav to have the right height (wrapped) on mobile.  Probably a better way.